.storybook/main.js

```diff
@@ -25,4 +25,8 @@ module.exports = {
   core: {
     builder: 'webpack5',
   },
+  // https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#emotion11-quasi-compatibility
+  features: {
+    emotionAlias: false,
+  },
```

Comment:
This is to resolve the `@emotion` import problems with Storybook. When the next major version of Storybook is released, we'll need to revisit if we still need this feature flag.